### ISSUE TITLE
Feature request - Close multiple cases

### ISSUE THREAD

**Issue by zappeee** [TheHive-Project/TheHive#405]
### Request Type
Feature Request

It would great if it was possible to close multiple cases at once.

**Comment by nadouani** [comment]
Hello,

the issue with closing task in a bulk way is that, closing a case requires user interaction: add a summary, specify resolution status and impact, and setting metrics and closing tasks.

What could be potentially done is:
1/ select a bunch of cases
2/ open the close dialog for the selected cases, one by one

This will allow the user to close some cases without having to navigate to case details page.

Is this what you expect?

**Comment by zappeee** [comment]
I see that I forgot to mention that I was talking about similar/duplicate cases.
In that case I don't think there's a need to get the close dialog for each and every case. Just one would do.

**Comment by saadkadhi** [comment]
@zappeee if there are similar or dups, merge them beforehand into one and close the new case created from your merged cases using the normal close dialog.

**Comment by zappeee** [comment]
@saadkadhi, that's unfortunately as time consuming as closing them one by one as I can't merge multiple cases.
Also it also creates new cases so, no offense, it looks all messed up.

**Comment by saadkadhi** [comment]
@zappeee well when you do merges, you will only have to close a single case at the end (thus filling the status, summary, metrics etc.). 

I am afraid we can't implement the feature you are looking for as this will twist the logic of the application in a serious fashion: if an analyst opens a case by error or create a dup of an existing one, then they need to supply information either during the close dialog or by merging it with another case. If we offer the ability to close multiple cases at once without forcing the user to fill the close dialog, this will open the way to abuse and oversight, and quite the headache for reading the audit trail properly.

**Comment by zappeee** [comment]
@saadkadhi, I know but I find it very hard to follow the trail afterwards.

Maybe we are working in a unsupported way but we have scripted the creation of new cases so sometimes we get 20 similar alerts which opens 20 cases in TH. Then closing multiple cases would make sense for us.

**Comment by saadkadhi** [comment]
In alerts, you have the option to preview and, depending on similarities with existing cases, either drop the alert, create a new case __or__ merge it in an existing case. 

I am going to go ahead and close this feature request given the efforts it'd require and the modification of the app logic.
